jsapp/js/components/account/dataStorage.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+
+export default class DataStorage extends React.Component<
+  {}, {}
+> {
```

Comment:
I know this is just temporary, but if you have a component with no props and no state, you can do `extends React.Component {`